Onboarding/menu-non-personalised.md

```diff
@@ -0,0 +1,687 @@
+    ) do
+      text("@list_error_text")
+    end
+end
+
+card HealthGuide when is_nil_or_empty(profile_completion), then: PromptZero do
+  update_contact(topic: "@selected_topic")
+end
+
+card HealthGuide, then: PromptPartial do
+  update_contact(topic: "@selected_topic")
+end
+
+card ViewTopics, then: LibraryTopics do
+  update_contact(topic: "@selected_topic")
+end
+
+card HelpCentre, then: GoToTakeATour do
+  update_contact(topic: "@selected_topic")
+end
+
+card Profile, then: YourProfile do
+  update_contact(topic: "@selected_topic")
+end
+
+card ManageUpdate, then: ManageUpdates do
+  update_contact(topic: "@selected_topic")
+end
+
+card ManageData, then: ManageData do
+  update_contact(topic: "@selected_topic")
+end
+
+card TakeATour, then: GoToTakeATour do
+  update_contact(topic: "@selected_topic")
+end
+
+card About, then: AboutPrivacy do
+  update_contact(topic: "@selected_topic")
+end
+
+```
+
+## Zero Profile
+
+```stack
+card PromptZero, then: DisplayPromptZero do
+  search =
+    get(
+      "https://content-repo-api-qa.prk-k8s.prd-p6t.org/api/v2/pages/",
+      query: [
+        ["slug", "mnch_onboarding_personalisation_prompt_zero"]
+      ],
+      headers: [["Authorization", "Token @config.items.contentrepo_token"]]
+    )
+
+  page_id = search.body.results[0].id
+
+  content_data =
+    get(
+      "https://content-repo-api-qa.prk-k8s.prd-p6t.org/api/v2/pages/@page_id/",
+      query: [
+        ["whatsapp", "true"]
+      ],
+      headers: [["Authorization", "Token @config.items.contentrepo_token"]]
+    )
+
+  message = content_data.body.body.text.value
+  button_labels = map(message.buttons, & &1.value.title)
+end
+
+card DisplayPromptZero, then: DisplayPromptZeroError do
+  buttons(
+    DomainShowcase: "@button_labels[0]",
+    MainMenu: "@button_labels[1]"
+  ) do
+    text("@message.message")
+  end
+end
+
+card DisplayPromptZeroError, then: DisplayPromptZeroError do
+  buttons(
+    DomainShowcase: "@button_labels[0]",
+    MainMenu: "@button_labels[1]"
+  ) do
+    text("@button_error_text")
+  end
+end
+
+```
+
+```stack
+card DomainShowcase do
+  log("DomainShowcase")
+end
+
+card MainMenu do
+  log("MainMenu")
+end
+
+```
+
+## Prompt Partial Profile
+
+```stack
+card PromptPartial, then: DisplayPromptPartial do
+  search =
+    get(
+      "https://content-repo-api-qa.prk-k8s.prd-p6t.org/api/v2/pages/",
+      query: [
+        ["slug", "mnch_onboarding_personalisation_prompt_partial"]
+      ],
+      headers: [["Authorization", "Token @config.items.contentrepo_token"]]
+    )
+
+  page_id = search.body.results[0].id
+
+  content_data =
+    get(
+      "https://content-repo-api-qa.prk-k8s.prd-p6t.org/api/v2/pages/@page_id/",
+      query: [
+        ["whatsapp", "true"]
+      ],
+      headers: [["Authorization", "Token @config.items.contentrepo_token"]]
+    )
+
+  message = content_data.body.body.text.value
+  loading_message = substitute(message.message, "{@username}", "@contact.name")
```

Comment:
oof, we can't have @'s in the variable name like that. If we have a variable username, then it would possibly do something very strange. Please change the text in the sheet to `{username}`